modules/planning/common/reference_line_info.h

```diff
@@ -203,6 +204,8 @@ class ReferenceLineInfo {
 
   SLBoundary adc_sl_boundary_;
 
+  SLBoundary vehicle_sl_boundary_;
```

Comment:
what is the difference between adc_sl_boundary_ and vehicle_sl_boundary_?

Reply:
adc_sl_boundary is generated by the stitching point while the latter is generated by the realtime vehicle position

Reply:
can you add these comments for these two variables? thanks.